ale/drivers/dawn_drivers.py

```diff
@@ -223,3 +227,38 @@ def sensor_model_version(self):
           ISIS sensor model version
         """
         return 2
+
+    @property
+    def detector_center_sample(self):
+        """
+        Returns center detector sample acquired from Spice Kernels.
+        Expects ikid to be defined. This should be the integer Naid ID code for
+        the instrument.
+        Returns
+        -------
+        : float
+          center detector sample
+        """
+        return float(spice.gdpool('INS{}_BORESIGHT'.format(self.ikid), 0, 3)[0])
+
+    @property
+    def detector_center_line(self):
+        """
+        Returns center detector line acquired from Spice Kernels.
+        Expects ikid to be defined. This should be the integer Naid ID code for
+        the instrument.
+        Returns
+        -------
+        : float
+          center detector line
+        """
+        return float(spice.gdpool('INS{}_BORESIGHT'.format(self.ikid), 0, 3)[1])
+
+
+    @property
+    def detector_start_sample(self):
```

Comment:
I'm not sure if detector_start_sample, or detector_start_line is correct. Was the closest thing I could find in the dawn ik kernels that looked similar to messenger.

Reply:
I think they actually are just both 1?